File: internal/content/testdata/bench/csharp/nc-palindromic-substrings.cs
public class Solution {
    public long count_substrings(string s) {
        long count = 0;

        void Expand(int left, int right) {
            while (left >= 0 && right < s.Length && s[left] == s[right]) {
                count++;
                left--;
                right++;
            }
        }

        for (int i = 0; i < s.Length; i++) {
            Expand(i, i);
            Expand(i, i + 1);
        }

        return count;
    }
}

File: internal/content/testdata/bench/go/nc-palindromic-substrings.go
func count_substrings(s string) int {
	count := 0

	expand := func(left, right int) {
		for left >= 0 && right < len(s) && s[left] == s[right] {
			count++
			left--
			right++
		}
	}

	for i := range s {
		expand(i, i)
		expand(i, i+1)
	}

	return count
}
